.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,19 @@
+#Contributing
```

Comment:
Looks like this needs a space after `#` to correctly register as a header.

Also it might be nice to add a step between 6 & 7 to commit the local changes and push to their fork before creating the pull request (for clarity).